UI Test Runner/UITests/FormViewTests.swift

```diff
@@ -609,6 +613,16 @@ final class FormViewTests: XCTestCase {
     }
 }
 
+private extension Date.ParseStrategy {
+    /// A parse strategy for date/time strings with a yyyy-MM-dd'T'HH:mm:ss format.
+    static var arcGISDateParseStrategy: Self {
+        .fixed(
+            format: "\(year: .defaultDigits)-\(month: .defaultDigits)-\(day: .defaultDigits)T\(hour: .defaultDigits(clock: .twentyFourHour, hourCycle: .zeroBased)):\(minute: .defaultDigits):\(second: .defaultDigits)",
+            timeZone: .current
+        )
+    }
+}
```

Comment:
I tried to remove this and use the one in the Toolkit, but I couldn't get it to compile. I made the toolkit one internal, added `@testable import ArcGISToolkit` and `import Foundation` to this file, but kept getting this error:
```
Undefined symbol: _$s10Foundation13ParseStrategyP13ArcGISToolkitAA4DateVABVRszrlE010arcGISDatebC0AGvgZ
```

Reply:
Yes, I had tried this exact thing to prevent the repeated code myself and after a long build time it failed and I got the same error. You'd think it should be able to work.

Reply:
🤷

Reply:
This shouldn't be necessary. A test shouldn't try to duplicate exactly what the code being tested does. Correct me if I'm wrong, but the test is just using the strategy to compute the expected date from a string. Instead, just get the time interval of the calculated date and then replace the date calculation with a date created from that time interval. Or, if desired, create a date from date components. But unless I'm missing something, the test does not and should not be using a date parse strategy.